modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -219,6 +219,34 @@ public void testBoostFreshness() throws Exception {
 		_assertSearchIgnoreRelevance("[coca cola, pepsi cola]");
 	}
 
+	@Test
+	public void testBoostLongerContents() throws Exception {
+		_setUpJournalArticles(
+			new String[] {"alpha alpha", "", ""},
+			new String[] {"beta alpha", "charlie alpha", "alpha alpha"});
+
+		_updateElementInstancesJSON(
+			new Object[] {
+				HashMapBuilder.<String, Object>put(
+					"boost", 100
+				).put(
+					"factor", 1.5
+				).put(
+					"modifier", "ln"
+				).build()
+			},
+			new String[] {"Boost Longer Contents"});
+
+		_keywords = "alpha";
+
+		_assertSearch("[beta alpha, alpha alpha, charlie alpha]");
```

Comment:
This seems to indicate that we boosted shorter contents to go first. @gustavolimav, could you explain?

Reply:
@joshuacords 

`_setUpJournalArticles(
			new String[] {"alpha alpha", "", ""},
			new String[] {"beta alpha", "charlie alpha", "alpha alpha"});`

When I create the journal articles the only article with content is "beta alpha", so when we use the boost longer contents element, "beta alpha" is supposed to be the highest score, and if we don't use the element the article with the highest score is "alpha alpha".

Probably it's better to add another article with content to prove that it's really working.